src/pytest_cases/fixture_parametrize_plus.py

```diff
@@ -874,6 +876,7 @@ def _create_params_alt(fh, test_func, union_name, from_i, to_i, hook):  # noqa
                 return SingleParamAlternative.create(new_fixture_host=fh, test_func=test_func,
                                                      param_union_name=union_name, argnames=argnames, i=i,
                                                      argvalue=marked_argvalues[i], id=_id,
+                                                     scope=scope or "function",
```

Comment:
```suggestion
                                                     scope="function" if scope is None else scope,
```

Reply:
Done in 484743a61a3898e2a2d91e16db2713101d9dae74